models/articles-model.js

```diff
@@ -41,4 +41,36 @@ function selectArticles(){
     })
 }
 
-module.exports = {selectArticleById, selectArticles}
\ No newline at end of file
+function selectCommentsById (articleId){
+    return db.query(`
+    SELECT
+    comments.comment_id,
+    comments.votes,
+    comments.created_at,
+    comments.author,
+    comments.body,
+    comments.article_id
+    FROM 
+    comments
+    JOIN
+    articles ON comments.article_id = articles.article_id
+    WHERE comments.article_id = ${articleId}
+    GROUP BY
+    comments.comment_id,
+    comments.votes,
+    comments.created_at,
+    comments.author,
+    comments.body,
+    comments.article_id
+    ORDER BY
+    comments.created_at DESC;`)
+    .then(({rows}) => {
+        if(rows.length !== 0){
+            return rows
+        }else {
+            return Promise.reject({ status: 404, message: 'Article does not exist' })
+        }
+    })
+}
+
```

Comment:
This is a very complex query, we should be able to request the comments for an article here using just the article_id as it is already included in the comment.